modulefiles/module_base.hera.lua

```diff
@@ -0,0 +1,35 @@
+help([[
+Load environment to run GFS on Hera
+]])
+
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+
+load(pathJoin("hpc", os.getenv("hpc_ver")))
+load(pathJoin("hpc-intel", os.getenv("hpc_intel_ver")))
+load(pathJoin("hpc-impi", os.getenv("hpc_impi_ver")))
+
```

Comment:
TCL version was loading hpss module

Reply:
Thanks for catching this! This is what I get for using the Orion one as the base for the new Hera one (Orion didn't have/need a HPSS module). Have added the `hpss` module into `module_base.hera.lua` and added `hpss_ver` to `hera.ver` @ 7425fd6